nautobot/docs/development/core/local-k8s.md

```diff
@@ -0,0 +1,126 @@
+# Using Local Kubernetes Cluster
+
+This guide will help you set up a local Kubernetes cluster using [Kind](https://kind.sigs.k8s.io/) and connect running Nautobot containers to the Kind network.
+
+## Kind Installation
+
+To install Kind on macOS:
+
+```bash
+brew install kind
+```
+
+To install Kind on Linux:
+
+```bash
+export KIND_VERSION=0.24.0
+curl -fsSL \
+    "https://kind.sigs.k8s.io/dl/v$KIND_VERSION/kind-linux-amd64" \
+    -o /usr/local/bin/kind
+chmod +x /usr/local/bin/kind
+```
+
+To create a new Kind cluster:
+
+```bash
+# Create the cluster
+kind create cluster \
+    --name=nautobot \
+    --config development/kind-config.yaml
+
+# Export the kubeconfig to access the cluster
+kind get kubeconfig --internal --name=nautobot > development/kind-kube-config
+```
+
+## Connecting Nautobot Containers
+
+This chapter will help you connect running Nautobot containers to the Kind network. It is necessary to do this after starting the Nautobot containers.
+
+Verify Nautobot containers are running:
+
+```bash
+# Start Nautobot containers if they are not running
+invoke start
+
+# Verify the containers are running
+docker ps
+```
+
+You should see something like this:
+
+```text
+CONTAINER ID   IMAGE                           COMMAND                  CREATED          STATUS          PORTS                    NAMES
+...
+1f191e104357   local/nautobot-dev:local-py3.11 "watchmedo auto-rest…"   2 minutes ago    Up 37 seconds (healthy)   0.0.0.0:8081->8080/tcp, [::]:8081->8080/tcp nautobot-celery_worker-1
+d396ed398a4c   local/nautobot-dev:local-py3.11 "/docker-entrypoint.…"   2 minutes ago    Up 2 minutes (healthy)    0.0.0.0:8080->8080/tcp, :::8080->8080/tcp nautobot-nautobot-1
+...
+```
+
+Verify the Kind Docker network name is `kind`:
+
+```bash
+docker network ls
```

Comment:
Perhaps `docker network ls --filter name=kind`?

Reply:
Kept as is, IMHO it's better for the user to see all his Docker networks.